x-pack/plugins/siem/public/components/page/network/tls_table/columns.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TlsNode } from '../../../../graphql/types';
+import { Columns } from '../../../load_more_table';
+
+import * as i18n from './translations';
+import { getRowItemDraggables, getRowItemDraggable } from '../../../tables/helpers';
+
+export const getTlsColumns = (
+  tableId: string
+): [
+  Columns<TlsNode>,
+  Columns<TlsNode>,
+  Columns<TlsNode['_id']>,
+  Columns<TlsNode>,
+  Columns<TlsNode>
+] => [
+  {
+    field: 'node',
+    name: i18n.ISSUER,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: false,
+    render: ({ _id, issuerNames }) =>
+      getRowItemDraggables({
+        rowItems: issuerNames,
+        attrName: 'tls.server_certificate.issuer.common_name',
+        idPrefix: `${tableId}-${_id}-table-issuerNames`,
+      }),
+  },
+  {
+    field: 'node',
+    name: i18n.SUBJECT,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: false,
+    render: ({ _id, alternativeNames, commonNames }) =>
+      alternativeNames != null && alternativeNames.length > 0
+        ? getRowItemDraggables({
+            rowItems: alternativeNames,
+            attrName: 'tls.server_certificate.alternative_names',
+            idPrefix: `${tableId}-${_id}-table-alternative-name`,
+          })
+        : getRowItemDraggables({
+            rowItems: commonNames,
+            attrName: 'tls.server_certificate.subject.common_name',
+            idPrefix: `${tableId}-${_id}-table-common-name`,
+          }),
+  },
+  {
+    field: 'node._id',
+    name: i18n.SHA1_FINGERPRINT,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: true,
+    render: sha1 =>
+      getRowItemDraggable({
+        rowItem: sha1,
+        attrName: 'tls.server_certificate.fingerprint.sha1',
+        idPrefix: `${tableId}-${sha1}-table-sha1`,
+      }),
+  },
+  {
+    field: 'node',
+    name: i18n.JA3_FINGERPRINT,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: false,
+    render: ({ _id, ja3 }) =>
+      getRowItemDraggables({
+        rowItems: ja3,
+        attrName: 'tls.fingerprints.ja3.hash',
+        idPrefix: `${tableId}-${_id}-table-ja3`,
+      }),
+  },
+  {
+    field: 'node',
+    name: i18n.VALID_UNTIL,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: false,
+    render: ({ _id, notAfter }) =>
+      getRowItemDraggables({
+        rowItems: notAfter,
+        attrName: 'tls.server_certificate.not_after',
+        idPrefix: `${tableId}-${_id}-table-notAfter`,
```

Comment:
I think we want to format this date as per the user defined date format in the Kibana Advance Settings? We've been using this for all non-relative dates if I recall correctly.

So instead of:
<img src="https://user-images.githubusercontent.com/2946766/56867158-58150580-699f-11e9-8415-561a02b8f80f.png" width="200">

it would look like:
<img src="https://user-images.githubusercontent.com/2946766/56867386-c5c23100-69a1-11e9-8d1b-3712b6d0d2d6.png" width="200">

Using this custom renderer:
``` typescript
render: ({ _id, notAfter }) =>
  getRowItemDraggables({
    rowItems: notAfter,
    attrName: 'tls.server_certificate.not_after',
    idPrefix: `${tableId}-${_id}-table-notAfter`,
    render: validUntil => (
      <LocalizedDateTooltip date={moment(new Date(validUntil)).toDate()}>
        <PreferenceFormattedDate value={new Date(validUntil)} />
      </LocalizedDateTooltip>
    ),
  }),
```

Reply:
Thanks for the free code 🐟, implemented it.